RedirectApplication/RedirectApplication/Controllers/ValuesController.cs

```diff
@@ -36,12 +37,16 @@ public HttpResponseMessage Get()
         // GET api/values/5
         public string Get(int id)
         {
+            
             return "value";
         }
 
         // POST api/values
         public void Post([FromBody]string value)
         {
+            string tree = "{\r\n   \"Conditions\": \r\n  [\r\n    {\r\n      \"name\": \"Composite\",\r\n      \"Rules\": [\r\n        {\r\n          \"name\": \"ByBrowser\",\r\n          \"Browser\": \"Safari\"\r\n        },\r\n        {\r\n          \"name\": \"ByIp\",\r\n          \"Ip\": [ \"192.168.0.1\", \"192.168.1.192\" ]\r\n        }\r\n      ],\r\n      \"url\": \"ya.ru\",\r\n    },\r\n    {\r\n      \"name\": \"ByBrowser\",\r\n      \"Browser\": \"Chrome\",\r\n      \"url\": \"vk.com\"\r\n    }\r\n  ]\r\n}";
+            JsonWorking json = new JsonWorking();
+            json.Main(tree);
```

Comment:
It should be added in another ticket PR then. History should be clean to other current/future developers, adding unrelated/dead code can cause many confusions in the future. Less code you ship to PR - less comments you've get ;)